.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig <http://EditorConfig.org>
+root = true
+
+# elementary defaults
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = tab
+indent_style = space
+insert_final_newline = true
+max_line_length = 80
```

Comment:
Most if not all other elementary repositories have 80 as the editorconfig max_line_length.  I think this is so Code puts the line width *guide* at 80 characters  I presume this is so code lines (not including comments?) should endeavour to be shorter than this for readability whereas 120 is a hard limit that triggers a lint error unless overridden.

If I am wrong and this needs to be updated then it will need to be done in many other repositories.